corim/extensions.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+package corim
+
+import (
+	"github.com/veraison/corim/extensions"
+)
+
+type IEntityValidator interface {
+	ValidEntity(*Entity) error
+}
+
+type ICorimValidator interface {
+	ValidCorim(*UnsignedCorim) error
+}
+
+type ISignerValidator interface {
+	ValidSigner(*Signer) error
+}
+
+type Extensions struct {
```

Comment:
This is extremely confusing to the reader.

You have same name spread everywhere.

Here inside `corim/extensions.go`
Then inside extensions/extensions.go
and same for comid/extensions.go

Perhaps you should consider, calling corim/extensions.go as

type CoRIMExtensions struct {
       extensions.Extensions
}